src/pages/home/report/ReportActionItem.js

```diff
@@ -623,6 +624,12 @@ function ReportActionItem(props) {
         );
     }
 
+    // For the `pay` IOU action on non-send money flow we don't want to show anything
+    // as the preview action is already shown by the create IOU action above
+    if (props.action.actionName === CONST.REPORT.ACTIONS.TYPE.IOU && originalMessage && originalMessage.type === CONST.IOU.REPORT_ACTION_TYPE.PAY && !isSendingMoney) {
+        return null;
+    }
```

Comment:
Discussing [here in Slack](https://expensify.slack.com/archives/C07J32337/p1697843476404399?thread_ts=1697843073.603539&cid=C07J32337), but I thought we agreed that this would not eb rendered if the `isWaitingOnBankAccount` property is true (so we only show the enable wallet flow). side note, it still shows `paid $10` in the LHN as the pay report action is here so we would need to add the same logic to LHN.

So this should still be rendered when `isWaitingOnBankAccount` is false hence when we actually sent the payment

Reply:
> Discussing [here in Slack](https://expensify.slack.com/archives/C07J32337/p1697843476404399?thread_ts=1697843073.603539&cid=C07J32337), but I thought we agreed that this would not eb rendered if the isWaitingOnBankAccount property is true

Oops, I missed the fact that we show a system message for pay IOU action, I thought we haven't still not figure out the use case for pay IOU action so I have hide that completely. I will get this fix ASAP. Thanks for pointing it out.